.ci/actions/main-sync/action.yml

```diff
@@ -0,0 +1,115 @@
+name: 'Sync main branch'
+description: 'Create a PR to sync main branch with main-apache excluding '
+
+inputs:
+  username:
+    description: 'Username for git'
+    required: false
+    default: kie-ci
+  useremail:
+    description: 'User email for git'
+    required: false
+    default: kie-ci0@redhat.com
+  main_sync_workflow_exclude_paths:
+    description: 'Paths to be excluded during merge'
+    required: false
+    default: .ci .github
+  main_sync_workflow_pr_reviewers:
+    description: 'Reviewers for the PR'
+    required: false
+    default: ''
+  dry_run:
+    description: 'True to perform a dry run (dry run git push and skip create PR step)'
```

Comment:
```suggestion
    description: 'True to perform a dry run (dry run will skip git push and PR creation)'
```

Reply:
The dry run doesn't really skip the `git push` but it runs it with the `--dry-run` option.
I didn't do the same for the `gh pr create` command as the official doc says it "may push some changes":
```
 --dry-run 
    Print details instead of creating the PR. May still push git changes.
```
https://cli.github.com/manual/gh_pr_create

Reply:
I see, ignore my suggestion then.